src/transaction/log_applier.h

```diff
@@ -52,6 +52,12 @@ typedef enum
   REPL_FILTER_EXCLUDE_TBL
 } REPL_FILTER_TYPE;
 
+typedef enum
```

Comment:
This is a header of old HA replication implementation; better to move the enum in one of new system headers.